client/app/dedicatedCloud/datacenter/drp/steps/ovh/mainPcc/index.js

```diff
@@ -19,8 +19,8 @@ angular
         },
       })
       .state('app.dedicatedClouds.datacenter.drp.ovh.mainPccStep.legacyOrderIp', {
-        controller: 'IpOrderCtrl',
-        templateUrl: 'ip/ip/order/ip-ip-order.html',
+        controller: 'IpLegacyOrderCtrl',
+        templateUrl: 'ip/ip/legacyOrder/ip-ip-legacyOrder.html',
```

Comment:
Could those be imported instead ? :)

Reply:
As seen together, importing the template would result in a `../../../../../ip/ip ...` so let's keep it like this